File: sdks/php/src/Providers/Google.php
<?php

declare(strict_types=1);

namespace CyberArmor\Providers;

use CyberArmor\Client;

final class Google extends OpenAICompatible
{
    public function __construct(Client $client, object $openai, string $tenantId, string $model = 'gemini-2.0-flash')
    {
        parent::__construct($client, $openai, $tenantId, 'google', $model);
    }
}

File: sdks/ruby/lib/cyberarmor/providers/google.rb
# frozen_string_literal: true

module CyberArmor
  module Providers
    class Google < OpenAICompatible
      def initialize(client:, openai:, tenant_id:, model: 'gemini-2.0-flash')
        super(client: client, openai: openai, tenant_id: tenant_id, provider: 'google', model: model)
      end
    end
  end
end

